tests/test_staticfiles.py

```diff
@@ -20,6 +23,27 @@ def test_staticfiles(tmpdir):
     assert response.text == "<file content>"
 
 
+def test_staticfiles_head_with_middleware(tmpdir):
```

Comment:
I guess it'd be worth linking to this issue in the docstring, since it's not obvious what the point of this test is otherwise.

Reply:
done